client/src/app.d.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-empty-interface */
 /// <reference types="@sveltejs/kit" />
 
```

Comment:
How will the used types that should be defined here, interact with the types defined in mocks? Or there won't be any mock types and all types will be defined here?

Reply:
It seems logical to move the request-response mock types into the app src,
it would give us a strict interface to follow.

Although, I am not sure where to move them, maybe I leave it as is for now.

Reply:
app.d.ts should contain or import and re-export all necessary types I think, but yeah, leave it as is, when it starts to bother us, then we will move it 😄